delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -167,8 +167,11 @@ def _prepare_customer(self, picking):
         if picking.picking_type_id.code != "outgoing":
             partner = picking.partner_id
 
+        partner_name = partner.name
+        if not partner_name:
```

Comment:
Can't we write `if not partner.name`? (avoiding the `partner_name` used here and below).

Reply:
I'm not sure what's the point here.
The idea is to avoid fetching data from the record twice, as intuitively, i'd say it's slower than storing the value in a variable and then reusing it.

Reply:
Odoo already caches all basic data as soon as you access one of its field:
```python
partner.name    # First call: generate relevant queries to fetch data and store them in the cache
partner.name    # Second call: no queries generated, it simply returns the cached value (unless you called `clear_caches` meanwhile)
```
So a variable is useless here, it brings only noise.

Reply:
done